lib/install.js

```diff
@@ -218,7 +208,11 @@ function install (gyp, argv, callback) {
         })
 
         // start unzipping and untaring
-        req.pipe(gunzip).pipe(extracter)
+        res.pipe(tar.extract({
+          strip: 1,
+          cwd: devDir,
+          filter: isValid
+        }).on('close', _ => setTimeout(afterTarball, 1000)).on('error', cb))
```

Comment:
tar@3 does not support node 0.x.  Neither does npm@5.

I thought they'd fallen off the LTS window.  How long are they going to be nominally supported?

The timeout is a bit of debugging scaffolding that I neglected to remove.  Good catch, thanks.  I'll rebase that out right now.